packages/graphql-language-service-server/src/GraphQLCache.js

```diff
@@ -303,15 +303,19 @@ export class GraphQLCache implements GraphQLCacheInterface {
   ) => (result: Object) => {
     if (result.files && result.files.length > 0) {
       const graphQLFileMap = this._graphQLFileListCache.get(rootDir);
-      if (!graphQLFileMap) {
-        return;
-      }
       result.files.forEach(async ({ name, exists, size, mtime }) => {
+        const filePath = path.join(result.root, result.subscription, name);
+        if (projectConfig.schemaPath && filePath === projectConfig.schemaPath) {
```

Comment:
Is the schemaPath guaranteed to be absolute? Otherwise this condition might be false, even though it should be true.

Reply:
It gets resolved into an absolute path. My test uses a relative path in the config file to verify this works